samples/highcharts/accessibility/internationalization/demo.js

```diff
@@ -0,0 +1,250 @@
+                exporting: {
+                    chartMenuLabel: 'Diagram meny',
+                    exportRegionLabel: 'Diagram meny',
+                    menuButtonLabel: 'Se diagram meny'
+                }
+            },
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Later­...',
+            months: ['Januar', 'Februar', 'Mars', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Desember'],
+            shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Des'],
+            weekdays: ['Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag', 'Søndag­'],
+            exportButtonTitle: 'Eksporter',
+            printButtonTitle: 'Print',
+            printChart: 'Skriv ut diagram',
+            viewFullscreen: 'Se i fullskjerm',
+            rangeSelectorFrom: 'Fra',
+            rangeSelectorTo: 'Til',
+            rangeSelectorZoom: 'Zoom',
+            downloadPNG: 'Last ned som PNG-bilde',
+            downloadJPEG: 'Last ned som JPEG-bilde',
+            downloadPDF: 'Last ned som PDF-dokument',
+            downloadSVG: 'Last ned som SVG-bilde',
+            title: 'AAPL Aksjekurs'
+        }
+    },
+    de: {
+        lang: {
+            accessibility: {
+                rangeSelector: {
+                    maxInputLabel: 'Endpunkt auswählen',
+                    minInputLabel: 'Startpunkt auswählen'
+                },
+                chartContainerLabel: 'Interaktives Highcharts-Diagramm',
+                exporting: {
+                    chartMenuLabel: 'Diagramm-Menü',
+                    exportRegionLabel: 'Diagramm-Menü',
+                    menuButtonLabel: 'Diagramm-Menü ansehen'
+                }
+            },
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Daten werden geladen...',
+            months: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember'],
+            weekdays: ['Sonntag', 'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag'],
+            shortMonths: ['Jan', 'Feb', 'Mrz', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez'],
+            exportButtonTitle: "Exportieren",
+            printButtonTitle: "Drucken",
+            printChart: 'Diagramm ausdrucken',
+            viewFullscreen: 'Im Vollbild anschauen',
+            rangeSelectorFrom: "Von",
+            rangeSelectorTo: "Bis",
+            rangeSelectorZoom: "Zeitraum",
+            downloadPNG: 'Download als PNG-Bild',
+            downloadJPEG: 'Download als JPEG-Bild',
+            downloadPDF: 'Download als PDF-Dokument',
+            downloadSVG: 'Download als SVG-Bild',
+            resetZoom: "Zoom zurücksetzen",
+            resetZoomTitle: "Zoom zurücksetzen",
+            title: 'AAPL Aktienkurs'
+        }
+    }
+};
+
```

Comment:
We should ensure that the chart is also translated fully to screen reader users. There are a few more lang options to translate:
https://api.highcharts.com/highcharts/lang.accessibility.chartContainerLabel
https://api.highcharts.com/highcharts/lang.accessibility.svgContainerLabel
https://api.highcharts.com/highcharts/lang.accessibility.screenReaderSection.beforeRegionLabel
https://api.highcharts.com/highcharts/lang.accessibility.screenReaderSection.endOfChartMarker

And then there are a couple other options that we need to set differently for each language:
https://api.highcharts.com/highcharts/accessibility.series.descriptionFormatter
This describes each data series. It is probably easiest to just have this return `series.options.accessibility.description` and set that option to be "AAPL stock, series 1 of 2." and "AAPL stock navigator overview, series 2 of 2." - and translate these.

https://api.highcharts.com/highcharts/accessibility.screenReaderSection.beforeChartFormat 
Just replace all of this with a short description in each language explaining that it is a financial line chart showing AAPL stock prices, with an additional navigator series showing an overview of the stock.